core/src/main/proto/wallet.proto

```diff
@@ -90,7 +90,7 @@ message Key {
     ORIGINAL = 1;
 
     /** Encrypted with Scrypt and AES - Original bitcoin secp256k1 curve */
```

Comment:
I think this comment should be changed too, like you did in your other PR. And maybe mention that the details of the encryption are declared via `Wallet.EncryptionType`.